modules/tensor_mechanics/src/materials/TwoStressPlasticity.C

```diff
@@ -4,17 +4,23 @@
 /*          All contents are licensed under LGPL V2.1           */
 /*             See LICENSE for full restrictions                */
 /****************************************************************/
-#include "PQPlasticModel.h"
+#include "TwoStressPlasticity.h"
```

Comment:
OK, fair enough. How about "TwoParameterPlasticity"?  That's another term that seems to be used pretty widely.   (Here's a Wikipedia article that refers to a "three-parameter" model: https://en.wikipedia.org/wiki/Willam-Warnke_yield_criterion).

I just haven't ever heard these models referred to as two (or three) stress models, and a quick web search turns up nothing on that.